apps/io-wallet-user-func/src/infra/http/handlers/__test__/create-wallet-attestation.spec.ts

```diff
@@ -193,7 +188,7 @@ describe("CreateWalletAttestationHandler", async () => {
             isRevoked: true,
             revokedAt: new Date(),
             signCount: 0,
-            userId: "123" as NonEmptyString,
+            userId: "AAA" as FiscalCode,
```

Comment:
```suggestion
            userId: "AAACCC94E17H501P" as FiscalCode,
```

Or better
`const mockFiscalCode = "AAACCC94E17H501P"`